website/src/components/HomePageFeatures/HomepageFeatures.module.css

```diff
@@ -9,7 +9,7 @@
 }
 
 .featureImage {
-  text-align: center;
+  text-align:center;
```

Comment:
please revert this change

Reply:
i have reverted this change, please check the commit